kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/ResolverAAImpl.kt

```diff
@@ -480,8 +483,18 @@ class ResolverAAImpl(
         }
     }
 
-    // TODO: handle @JvmName annotations, mangled names
+    // TODO: handle library symbols
     override fun getJvmName(accessor: KSPropertyAccessor): String? {
+        (
+            (accessor.receiver.closestClassDeclaration() as? KSClassDeclarationImpl)
+                ?.ktClassOrObjectSymbol?.psi as? KtClassOrObject
+            )?.toLightClass()?.allMethods?.filterIsInstance<SymbolLightSimpleMethod>()
+            ?.singleOrNull {
+                (it.parameters.isNotEmpty() xor (accessor is KSPropertyGetter)) &&
```

Comment:
This line is very tricky, the idea is we don't have a way to tell if a light method is getter or setter, and both getter and setter shares a same property declaration as the `kotlinOrigin`, so we just check if an accessor has any parameters to determine whether it is a getter or setter.